extensions/amp-a4a/amp-a4a-internal.md

```diff
@@ -19,8 +19,8 @@ limitations under the License.
 <table>
```

Comment:
Same comment as above, should we rename the file and directory?

Reply:
I'd say no. Breaks links and the extension itself is still a4a.